### PR TITLE
rework system-restore and some other minor enhancements

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: not-supported/system-restore.sh [hunk 1]
```diff
@@ -430,108 +431,25 @@ done
...
+if ! yesno_box_yes "Do you want to restore Nextclouds previews? This might slow down the restore process by a lot.
+If you select 'No', the preview folder will be excluded from the restore process which can lead to preview issues in Nextcloud."
 then
-    msg_box "Something failed while performing the system-partition dry-run."
-    umount /tmp/borgsystem
-    umount /tmp/borgboot
-    umount /tmp/borgncdata &>/dev/null
-    restore_original_state
-    exit 1
+    PREVIEW_EXCLUDED="--exclude=/appdata_"*/preview/
```
[the remark this answers is not in the document]
Comment (enoch85) [reply not-supported/system-restore.sh:440]: @szaimen Have you seen this?

[the remark this answers is not in the document]
Comment (szaimen) [reply not-supported/system-restore.sh:440]: Yes, I still need to address this

[the remark this answers is not in the document]
Comment (szaimen) [reply not-supported/system-restore.sh]: done

**Comment by szaimen** [comment]
This is ready 👍

**Pull request merged**